pkgs/top-level/release.nix

```diff
@@ -77,6 +77,7 @@ let
               jobs.tests.cc-wrapper-libcxx-39.x86_64-darwin
               jobs.tests.stdenv-inputs.x86_64-darwin
               jobs.tests.macOSSierraShared.x86_64-darwin
+              jobs.tests.patch-shebangs.x86_64-darwin
```

Comment:
Do I need to list these out? I was just following the pattern but it feels sort of redundant

Reply:
We could use `[] ++ builtins.getAttrValues jobs.foo` everywhere instead. However the unstable jobset would update if there's an evaluation error for one of the platforms, instead of making the job disappear.